addons/purchase/test/average_price.yml

```diff
@@ -47,7 +47,7 @@
 - 
   !python {model: product.product, id: product_average_icecream}: |
```

Comment:
make the xml_id consistent with the cost asigned

Reply:
we have a task to get rid of yml files, we'll fix this one later on